android/src/main/java/vn/luongvo/kmm/survey/android/ui/screens/login/LoginViewModel.kt

```diff
@@ -1,33 +1,47 @@
 package vn.luongvo.kmm.survey.android.ui.screens.login
 
 import androidx.lifecycle.viewModelScope
-import kotlinx.coroutines.flow.catch
-import kotlinx.coroutines.launch
-import timber.log.Timber
+import kotlinx.coroutines.flow.*
 import vn.luongvo.kmm.survey.android.ui.base.BaseViewModel
-import vn.luongvo.kmm.survey.data.local.datasource.TokenLocalDataSource
+import vn.luongvo.kmm.survey.android.ui.navigation.AppDestination
+import vn.luongvo.kmm.survey.domain.usecase.IsLoggedInUseCase
 import vn.luongvo.kmm.survey.domain.usecase.LogInUseCase
 
 class LoginViewModel(
-    private val logInUseCase: LogInUseCase,
-    private val tokenLocalDataSource: TokenLocalDataSource
+    private val isLoggedInUseCase: IsLoggedInUseCase,
```

Comment:
@luongvo I think it should be better if we name the usecase with action prefix. what do you think?
```suggestion
    private val checkLoggedInUseCase: CheckLoggedInUseCase,
```

Reply:
@doannimble I think `IsLoggedInUseCase` fits my context here to return true or false. `Check` is value to know what is false or what is true, right? I think a function could start with a `verb` or a `is` word for a boolean value returning.